app/views/restaurants/edit.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+  <%- if @restaurant.errors.present? -%>
+    <div class="large-12 alert-box alert" data-alert>
+      <ul>
+        <%- @restaurant.errors.full_messages.each do |error| -%>
+          <li><%= error %></li>
+        <%- end -%>
+      </ul>
+    </div>
+  <%- end -%>
```

Comment:
This should preferably be taken care of by flashing the error messages from the controller and displaying them via the layout